ANSWERS.md

```diff
@@ -0,0 +1,15 @@
+What are PropTypes used for?
+
+A: Proptypes are used for validation of a given datatype. It's to 'type-check' your code to make sure you aren't passing the wrong datatype into the wrong componnent. For example if you had some type of text, you'd want to put proptype.string to ensure the right information is being passed through the component.
+
+What is a lifecycle event in React?
+
+A: The first thing is the creation of the component, then any added 'state' or 'props' will change/update the component, then comes the 'unmounting' of the component. This is a very simplified explaination, but I think it touches on the major themes of the react lifecycle.
+
+What is a Higher Order Component?
+
+A: A higher order component is not actually a 'component' per se, it's actually a function as I understand it. HOCs are used to avoid reapting code by giving us a way to share functionality shared by various components by taking in a component and returning a new component with added data.
+
+What are three different ways to style components in React?
+
```

Comment:
you can use styled components, vanilla css/preprocessors, or another library like reactstrap/bootstrap